CMakeLists.txt

```diff
@@ -347,8 +347,8 @@ endif()
 # Configuring the ASPECT targets themselves
 # ##############################################################################
 
-message(STATUS "")
-message(STATUS "===== Configuring ASPECT build targets =============")
+message("")
+message("===== Configuring ASPECT build targets =====")
```

Comment:
The number of = signs on the right is chosen so that the line lengths of all of these lines are equal. This produces a nice "block" layout, rather than ragged right.

Reply:
well, this is what main looks like for me:
```
-- ====================================================
-- ============ Configuring ASPECT ====================
-- ====================================================
-- ===== Configuring external libraries ===============
-- ===== Setting up Geodynamic World Builder version 0.6.0 in Debug mode
-- ===== Setting up Geodynamic World Builder version 0.6.0 in Release mode
-- ===== Configuring ASPECT build targets =============
-- ===== Configuring ASPECT documentation =============
-- ===== Writing final configuration =============
```

Your GWB lines can not be blocked correctly, and the last entry is incorrect.